src/pages/account-menu/in-a-drawer/Drawer.tsx

```diff
@@ -0,0 +1,178 @@
+import WebAssetIcon from '@material-ui/icons/WebAsset';
+
+const backgroundImage = require('../../../assets/cubes_tile.png').default;
+const linearGradientOverlayImage = `linear-gradient(to right, rgba(0, 123, 193, 1) 22.4%, rgba(0, 123, 193, 0.2) 100%), url(${backgroundImage})`;
+
+const useStyles = makeStyles((theme: Theme) => ({
+    avatarSize: {
+        height: '48px',
+        width: '48px',
+    },
+    backgroundGradient: {
+        backgroundImage: `${linearGradientOverlayImage}`,
+        backgroundSize: 'contain',
+        backgroundPosition: 'right',
+    },
+    closeIcon: {
+        marginRight: `-${theme.spacing(2)}px`,
+        marginTop: `-${theme.spacing(4)}px`,
+    },
+    extendedHeader: {
+        width: '100%',
+        padding: '16px 16px 4px',
+    },
+    header: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        zIndex: 1,
+        alignItems: 'center',
+        width: '100%',
+        height: '100%',
+    },
+    headerDetails: {
+        paddingTop: '16px',
+        position: 'relative',
+    },
+    subtitle: {
+        marginTop: `-${theme.spacing(0.5)}px`,
+    },
+}));
+
+type DrawerProps = {
+    open: boolean;
+    drawerToggler: () => void;
+};
+
+const avatarImage = require('../../../assets/avatar_40.png').default;
+
+export const PxbDrawer = (props: DrawerProps): JSX.Element => {
+    const { open, drawerToggler } = props;
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const variant = 'temporary';
+    const [selected, setSelected] = useState('1');
+
+    const navGroupItems: NavItem[] = [
+        {
+            title: 'Dashboard',
+            itemID: '1',
+            icon: <Dashboard />,
+            onClick: (): void => setSelected('1'),
+        },
+        {
+            title: 'Notifications',
+            itemID: '2',
+            icon: <Notifications />,
+            onClick: (): void => setSelected('2'),
+        },
+        {
+            title: 'Locations',
+            itemID: '3',
+            icon: <LocationOnIcon />,
+            onClick: (): void => setSelected('3'),
+        },
+        {
+            title: 'Analytics',
+            itemID: '4',
+            icon: <AssessmentIcon />,
+            onClick: (): void => setSelected('4'),
+        },
+        {
+            title: 'Assets',
+            itemID: '5',
+            icon: <WebAssetIcon />,
+            onClick: (): void => setSelected('5'),
+        },
+    ];
+
+    const ExtendedDrawerHeader = (): JSX.Element => (
+        <div className={classes.extendedHeader}>
+            <div className={classes.header}>
+                <Avatar alt="Chima Thabani" src={avatarImage} classes={{ root: classes.avatarSize }} />
+                <IconButton
+                    data-cy="toolbar-menu"
+                    color={'inherit'}
+                    edge={'end'}
+                    classes={{ edgeEnd: classes.closeIcon }}
+                    onClick={drawerToggler}
```

Comment:
I would call this `toggleDrawer` just for the sake of keeping functions in the verb form